appengine/test/tc_base_image.rb

```diff
@@ -0,0 +1,18 @@
+require "minitest/autorun"
```

Comment:
If you do this from `Rake::TestTask` require args, then it won't be necessary to add to every file.